examples/example-ssl.rs

```diff
@@ -115,6 +125,7 @@ mod example {
             opts.optopt("", "ca-cert", "Specify the trusted CA certificates", "FILE");
             opts.optopt("", "client-cert", "Specify the client certificate", "FILE");
             opts.optopt("", "client-key", "Specify key for the client certificate", "FILE");
+            opts.optflag("", "verify-server-name", "Performing server name verification.");
```

Comment:
can we settle down to the term "hostname verification", i.e. `verify-hostname` or `verify-broker-hostname`? i think this is generally how the feature is referred to, so it'd be nice if'd be consistent.